src/App.vue

```diff
@@ -80,6 +80,18 @@ export default {
 </template>
 
 <style>
+html {
+  scrollbar-width: none;
+}
+
+html::-webkit-scrollbar {
```

Comment:
I'm surprised you have to do any vendor-specific directives here. @jtomeck, do you know if there's a better way to handle this within a Vue style section?

Reply:
I believe this was just best practice from what I could see. I'd be interested to see what @jtomeck has to say